File: sdk/php/src/Model/SliderCaptchaResponse.php
<?php

namespace HJTPX\Captcha\Model;

class SliderCaptchaResponse
{
    public $sessionId;
    public $imageUrl;
    public $puzzleUrl;
    public $hintUrl;
    public $shape;
    public $secretY;
    public $imageWidth;
    public $imageHeight;
    public $tolerance;

    public function __construct(array $data)
    {
        $this->sessionId = $data['session_id'] ?? null;
        $this->imageUrl = $data['image_url'] ?? null;
        $this->puzzleUrl = $data['puzzle_url'] ?? null;
        $this->hintUrl = $data['hint_url'] ?? null;
        $this->shape = $data['shape'] ?? null;
        $this->secretY = $data['secret_y'] ?? null;
        $this->imageWidth = $data['image_width'] ?? null;
        $this->imageHeight = $data['image_height'] ?? null;
        $this->tolerance = $data['tolerance'] ?? null;
    }
}

File: sdk/java/src/main/java/com/hjtpx/captcha/model/SliderCaptchaResponse.java
package com.hjtpx.captcha.model;

import com.fasterxml.jackson.annotation.JsonProperty;

public class SliderCaptchaResponse {
    @JsonProperty("session_id")
    private String sessionId;

    @JsonProperty("image_url")
    private String imageUrl;

    @JsonProperty("puzzle_url")
    private String puzzleUrl;

    @JsonProperty("hint_url")
    private String hintUrl;

    @JsonProperty("shape")
    private int shape;

    @JsonProperty("secret_y")
    private int secretY;

    @JsonProperty("image_width")
    private int imageWidth;

    @JsonProperty("image_height")
    private int imageHeight;

    @JsonProperty("tolerance")
    private int tolerance;

    public String getSessionId() {
        return sessionId;
    }

    public void setSessionId(String sessionId) {
        this.sessionId = sessionId;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }

    public String getPuzzleUrl() {
        return puzzleUrl;
    }

    public void setPuzzleUrl(String puzzleUrl) {
        this.puzzleUrl = puzzleUrl;
    }

    public String getHintUrl() {
        return hintUrl;
    }

    public void setHintUrl(String hintUrl) {
        this.hintUrl = hintUrl;
    }

    public int getShape() {
        return shape;
    }

    public void setShape(int shape) {
        this.shape = shape;
    }

    public int getSecretY() {
        return secretY;
    }

    public void setSecretY(int secretY) {
        this.secretY = secretY;
    }

    public int getImageWidth() {
        return imageWidth;
    }

    public void setImageWidth(int imageWidth) {
        this.imageWidth = imageWidth;
    }

    public int getImageHeight() {
        return imageHeight;
    }

    public void setImageHeight(int imageHeight) {
        this.imageHeight = imageHeight;
    }

    public int getTolerance() {
        return tolerance;
    }

    public void setTolerance(int tolerance) {
        this.tolerance = tolerance;
    }
}
